docs/modules/ROOT/pages/index.adoc

```diff
@@ -1,9 +1,11 @@
 = Devfile
 :description: Software Defined Development Environments
 
-Devfiles describe and define a cloud native development environment.
+Devfiles are YAML files that describe and define an environment in Che, including the source code and development components, like IDE tools and application runtimes. Che uses these devfiles to make a cloud workspace composed of multiple containers.
 
-Devfiles are consumed by tools that transform the definition into a cloud workspace composed of multiple cloud native resources. Any number of identical workspaces can be created from the same devfile. They can be shared in various ways including a devfile registry or together with the source code of a project.
+Within your devfile workspace, you can create multiple projects. In the components of the project, like the editor, plugin, or environment, the devfile commands build, test, and run your application. Also, any number of workspaces can be created from the same devfile. This way, workspaces can be shared and worked on by different teams. So now you can easily transfer, collaborate and expand your work.
```

Comment:
@jc-berger:

* Unless necessary, avoid using "you", "your", or other personal pronouns. There's no need for a devfile, projects, or workspaces to belong to the reader.
* Use "plug-in" (see _The IBM Style Guide_ pg. 353).
* Use the serial comma (see _The IBM Style Guide_ pg. 241).
* Avoid marketing speak, including "easily" or "now".
* Don't edit text just for the sake of editing it. In this case, the orig. first sentence was perfectly fine and better described the devfile concept.